datetime/src/main/scala/com/fortysevendeg/scalacheck/datetime/instances/jdk8.scala

```diff
@@ -7,14 +7,53 @@ package com.fortysevendeg.scalacheck.datetime.instances
 
 import com.fortysevendeg.scalacheck.datetime.typeclasses._
 import java.time._
+import java.time.temporal.ChronoField
 import java.time.temporal.ChronoUnit.MILLIS
 
 trait Jdk8Instances {
-
-  implicit val jdk8ForDuration: ScalaCheckDateTimeInfra[ZonedDateTime, Duration] = new ScalaCheckDateTimeInfra[ZonedDateTime, Duration] {
+   
+  implicit val jdkForDurationZonedDateTime: ScalaCheckDateTimeInfra[ZonedDateTime, Duration] = new ScalaCheckDateTimeInfra[ZonedDateTime, Duration] {
     def addRange(zonedDateTime: ZonedDateTime, duration: Duration): ZonedDateTime = zonedDateTime.plus(duration)
     def addMillis(zonedDateTime: ZonedDateTime, millis: Long): ZonedDateTime = zonedDateTime.plus(millis, MILLIS)
     def getMillis(zonedDateTime: ZonedDateTime): Long = zonedDateTime.toInstant.toEpochMilli
+    override def isBefore(dt1: ZonedDateTime, dt2: ZonedDateTime): Boolean = dt1.isBefore(dt2)
+  }
+
+  // retained for compatibility
+  val jdk8ForDuration = jdkForDurationZonedDateTime
```

Comment:
Can this be completely removed? We're still so early on in the project that I don't mind if backwards compatibility is broken if it makes the project more coherent overall.